src/esp/bindings_js/bindings_js.cpp

```diff
@@ -153,20 +153,10 @@ EMSCRIPTEN_BINDINGS(habitat_sim_bindings_js) {
   em::class_<Sensor>("Sensor").function("specification",
                                         &Sensor::specification);
 
-  em::class_<SceneConfiguration>("SceneConfiguration")
-      .smart_ptr_constructor("SceneConfiguration",
-                             &SceneConfiguration::create<>)
-      .property("dataset", &SceneConfiguration::dataset)
-      .property("id", &SceneConfiguration::id)
-      .property("filepaths", &SceneConfiguration::filepaths)
-      .property("sceneUpDir", &SceneConfiguration::sceneUpDir)
-      .property("sceneFrontDir", &SceneConfiguration::sceneFrontDir)
-      .property("sceneScaleUnit", &SceneConfiguration::sceneScaleUnit);
-
   em::class_<SimulatorConfiguration>("SimulatorConfiguration")
       .smart_ptr_constructor("SimulatorConfiguration",
                              &SimulatorConfiguration::create<>)
-      .property("scene", &SimulatorConfiguration::scene)
```

Comment:
Can't we not add a dummy property here as a deprecated shim?

Reply:
I could, but I wanted things to break so that folks could find and fix any problems. Besides, it would have to be an object, because code refs to this follow the form scene.id so just having some kind of dummy here wouldn't work easily.

Reply:
Just to revisit, this could temporarily unblock a lot of the pain you are having right now with the python bindings.

Reply:
I do not see how.  Originally scene represented the struct (SceneConfiguration) that this PR is removing, which had as a member the string "id", while now it is replaced with "scene_id" which represents a string. Assignment originally was made to scene.id.  Not sure how i could dummy this in any acceptable way.

Reply:
Python is wild: https://github.com/facebookresearch/habitat-sim/compare/Remove_SceneConfiguration-backwards-compat?expand=1#diff-08c325c91ce6aa20a6124d796c72d68934251a35f1cbc6e24421e5c23617de69

If we do want to add a backwards compatibility layer here, we should do it after we have verified that our code doesn't need it tho.

Reply:
Thanks Erik.  I like that deprecation protocol.  If I'm still having problems with this change against hab-lab, I'll make use of it.